test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.ConnectionLifecycle.cs

```diff
@@ -91,16 +98,16 @@ IConnection ConnectionFactory()
                 });
             }
 
-            [Fact]
+            [Fact(Skip = "This test would require hooks into the HubConnection to function again")]
```

Comment:
Because we don't `DisposeAsync` anymore, right?

Reply:
Correct.

Reply:
> Re-read your PR description. Should we just remove the test?

Yes, I just wanted to discuss it. I'll remove it.